test/gauge-test.ts

```diff
@@ -0,0 +1,30 @@
+import { Gauge } from '../src';
+import { expect } from 'chai';
+
+describe('Gauge', () => {
+  let gauge: Gauge;
+
+  beforeEach(() => {
+    gauge = new Gauge();
+  });
+  it('sets the gauge', () => {
+    const value = 55;
+    expect(gauge.set(value).get()!.value).equals(value);
+  });
+  it('increments and decrements values', () => {
+    expect(gauge.inc().get()!.value).equals(1);
```

Comment:
Even if I add the check like `expect(gauge.set(value).get()).not.undefined;` it will be not enough for TS to do not use `!`
So I guess for tests the best way is — less code and less checks :)